### PR TITLE
Viewports refactor

### PR DESCRIPTION
**Pull request by winkerVSbecks**
* Make top level viewports page that covers all three integrations
* Move storybook specific info to `/storybook/viewports`

### CODE REVIEW

File: src/content/snapshotOptions/viewports.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+layout: "../../layouts/Layout.astro"
+title: Viewports
+description: Configure Chromatic to test responsive UIs at various viewports and breakpoints.
+sidebar: { order: 2 }
+---
+
+# Viewports for responsive UIs with breakpoints
+
+UI components respond and adapt to different device widths. Chromatic streamlines the process for visual testing by enabling you to capture each test at multiple viewport sizes.
```
Comment (jonniebigodes) [inline src/content/snapshotOptions/viewports.mdx:10]: Small suggestion here, feel free to disregard
```suggestion
UI components respond and adapt to different device widths. Chromatic streamlines visual testing, enabling you to capture each test at multiple viewport sizes.
```

**Review by jonniebigodes — APPROVED** [review]
@winkerVSbecks, this is looking great; thanks for adding these changes 🙏 ! I left one small item for you to look into when you can.

I've approved it to unblock you.

One question, though: I was wondering if we shouldn't check Chromatic's blog posts related to this feature and see if there are no broken links that could be generated when these changes go live.

File: src/content/snapshotOptions/viewports.mdx [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+layout: "../../layouts/Layout.astro"
+title: Viewports
+description: Configure Chromatic to test responsive UIs at various viewports and breakpoints.
+sidebar: { order: 2 }
+---
+
+# Viewports for responsive UIs with breakpoints
+
+UI components respond and adapt to different device widths. Chromatic streamlines visual testing, enabling you to capture each test at multiple viewport sizes.
```
Comment (domyen) [inline src/content/snapshotOptions/viewports.mdx:10]:
```suggestion
UI components respond and adapt to different device widths. Chromatic streamlines visual testing, enabling you to capture each test at multiple viewport sizes. Viewports is compatible with Storybook, Playwright, and Cypress integrations.
```

File: src/content/troubleshooting/faq/chromatic-mobile-testing.mdx [hunk 3]
```diff
@@ -8,5 +8,6 @@ section: compatibility
 # Do you support mobile testing?
```
Comment (domyen) [inline src/content/troubleshooting/faq/chromatic-mobile-testing.mdx:8]:
```suggestion
# Do you support mobile testing for iOS or Android?
```

Matches more keywords in search

**Review by domyen — APPROVED** [review]

**Pull request merged**